client/src/components/Form/InputDisplay/InputDisplay.tsx

```diff
@@ -88,17 +103,23 @@ const InputDisplay = (props) => {
 
 InputDisplay.propTypes = {
     inputTitle: PropTypes.string.isRequired,
-    inputValue: PropTypes.string,
+    inputValue: PropTypes.string.isRequired,
     inputId: PropTypes.string.isRequired,
-    inputValueSave: PropTypes.func,
+    inputValueSave: PropTypes.func.isRequired,
     inputVisibility: PropTypes.func,
     inputVisible: PropTypes.bool.isRequired,
     inputType: PropTypes.string,
-    inputFieldAs: PropTypes.string,
+    inputFieldAsSelect: PropTypes.bool,
     optionsForSelect: PropTypes.array,
     textForSelect: PropTypes.array,
-    toDisable: PropTypes.instanceOf(Set),
+    toDisable: PropTypes.oneOf<DisableInput>([ DisableInput.Visibility, DisableInput.Edit ]),
     validation: PropTypes.object,
+    // validation: PropTypes.exact({
+    //     required: PropTypes.bool,
+    //     minLength: PropTypes.number,
+    //     maxLength: PropTypes.number,
+    //     pattern: PropTypes.string
+    // }),
```

Comment:
why is it commented out?

Reply:
Proptypes validation does not work in this case. Do not know the solution yet.
Comment reminds me the direction of last trial.

Reply:
okay :)